hub/admin/extend_user.py

```diff
@@ -310,3 +310,14 @@ def _get_message(self, singular: bool, grace_period: int) -> str:
             message += f'View <a href="{url}">trash.</a>'
 
         return mark_safe(message)
+
+    def save_model(self, request, obj, form, change):
+        if obj.is_superuser:
+            if config.SUPERUSER_AUTH_ENFORCEMENT:
+                if obj.has_usable_password() and not config.MFA_ENABLED:
+                    raise ValidationError(
+                        (
+                            'Superusers must use either MFA or SSO with an unusable password.'
```

Comment:
Could we simplify this to say something like 'Superusers with a usable password must enable MFA' but maybe ask around. That term makes a lot of sense to me, but maybe not the general public.